diff --git a/Source/RenderStream/Private/RenderStream.cpp b/Source/RenderStream/Private/RenderStream.cpp
index 7431e7a..7b3c575 100644
--- a/Source/RenderStream/Private/RenderStream.cpp
+++ b/Source/RenderStream/Private/RenderStream.cpp
@@ -39,6 +39,7 @@
 
 #include "RenderStreamLogOutputDevice.h"
 #include "RenderStreamStats.h"
+#include "GameFramework/DefaultPawn.h"
 
 #include <map>
 #include <string>
@@ -695,6 +696,16 @@ void FRenderStreamModule::OnBeginFrame()
         RootActor->GetConfigData()->StageSettings.AllViewportsOCIOConfiguration.bIsEnabled = true;
         RootActor->GetConfigData()->StageSettings.AllViewportsOCIOConfiguration.OCIOConfiguration = settings->OCIOConfig;
     }
+
+    // Hide all default pawns
+    if (m_isFirstFrame && GWorld)
+    {
+        TArray<AActor*> FoundDefaultPawns;
+        UGameplayStatics::GetAllActorsOfClass(GWorld, ADefaultPawn::StaticClass(), FoundDefaultPawns);
+        for (AActor* DefaultPawn : FoundDefaultPawns)
+            DefaultPawn->SetActorHiddenInGame(true);
+    }
+    m_isFirstFrame = false;
 }
 
 void FRenderStreamModule::OnModulesChanged(FName ModuleName, EModuleChangeReason ReasonForChange)
diff --git a/Source/RenderStream/Private/RenderStream.h b/Source/RenderStream/Private/RenderStream.h
index 1ea4a30..2f9d979 100644
--- a/Source/RenderStream/Private/RenderStream.h
+++ b/Source/RenderStream/Private/RenderStream.h
@@ -90,4 +90,5 @@ public:
     TSharedPtr<FRenderStreamLogOutputDevice, ESPMode::ThreadSafe> m_logDevice = nullptr;
     double m_LastTime = 0;
     bool m_gameInstanceStarted = false;
+    bool m_isFirstFrame = true;
 };
